collaborative/IOU/heat_index.ipynb

```diff
@@ -162,218 +165,426 @@
+    "\n",
+    "In the following cells, we'll do all of the computation above in a single go, to condense the number of cells to run. You don't need to modify anything, unless you made changes above. "
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "fa535bfe-5117-40c9-ba4e-33789d3555ce",
+   "id": "39dc6988-6107-42bb-af55-9585fe94e546",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
-    "histidx_proj_hour = histidx_proj_hour.isel(time = histidx_proj_hour.time.dt.month.isin(month_subset))"
+    "# counts the number of hours in each day above the heat index threshold\n",
+    "num_heatidx_projhours = (heatidx_proj_hour >= hi_threshold).resample(time='1D').sum()\n",
+    "num_heatidx_projhours.name = 'Hours per day above Heat Index threshold of {}°F'.format(hi_threshold)\n",
+    "\n",
+    "# sum per year\n",
+    "num_heatidx_proj_hours_per_year = num_heatidx_projhours.groupby('time.year').sum('time')\n",
+    "num_heatidx_proj_hours_per_year.name = 'Hours per year above Heat Index threshold of {}°F'.format(hi_threshold)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d6b58caa-4c1d-4dd5-a915-00fed9877366",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# visualize\n",
+    "num_heatidx_proj_hours_per_year.hvplot.line(x='year', by='simulation')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5f5f571d-a8c9-472a-8dd8-736136b3da12",
+   "metadata": {},
+   "source": [
+    "### 2c) Export counts of hours per year\n",
+    "First we'll calculate summary statistics for our data object with a handy function `compute_multimodel_stats`, which will provide the min, max, mean, and median. We'll also use a helper function `summary_table` which will transform our data objects into an easy to read dataframe table."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "4d20af18-a2d8-4707-b97b-ccf064bbfed0",
+   "id": "1a6e222b-dbeb-46c8-b712-7d101e940d39",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
-    "histidx_proj_hour = ck.load(histidx_proj_hour)\n",
-    "heatidx_proj_day = heatidx_proj_hour.resample(time='1D').max() # daily max"
+    "# calculate statistics\n",
+    "num_heatidx_hist_hours_per_year = compute_multimodel_stats(num_heatidx_hist_hours_per_year)\n",
+    "\n",
+    "# table format with summary stats\n",
+    "df_to_export = summary_table(num_heatidx_hist_hours_per_year)\n",
+    "df_to_export.head(5) # see first 5 rows"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ce7764ff-e21f-4329-aacc-5852bb84864d",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "fn = 'num_heatidx_hours_per_year_{}'.format(station_name.replace(\" \", \"_\").replace(\"(\", \"\").replace(\")\", \"\"))\n",
+    "df_to_export.to_csv('{}.csv'.format(fn))"
```

Comment:
Could consider adding text about where to view and download the `csv` file after the data has been exported to it.